docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|`size`
+|`number`
+|The font size in pixels 
+
+Default: `14`
+
+|`lHeight` 
+
+Alias: `lineHeight`
+|`number`, `null`
+|The line height in pixels
+
+|`family`
+|`string`
+|An acceptable CSS web font string 
+
+Default: `"'Open Sans', Helvetica, Arial, sans-serif"`
+
+|`color`
+|`string`
+|The text color
+
+|`weight`
+|`string`
+|The font weight, for example, `"normal"`, `"bold"`, `"bolder"`, `"lighter"`, 
+`"100"`, `"200"`, `"300"`, `"400"`, `"500"`, `"600"`, `"700"`, `"800"`, `"900"` 
+
+Default: `"normal"`
+
+|`underline`
+|`boolean`
+|Underline the text? 
+
+Default: `false`
+
+|`italic`
+|`boolean`
+|Italicize the text? 
+
+Default: `false`
+
+|`align`
+|`string`
+|The horizontal alignment of text 
+
+Default: `left`
+|===
+
+*Returns:* `style`
+
+
+[float]
+[[formatdate_fn]]
+=== `formatdate`
+
+Format a valid date string or number of milliseconds since epoch using https://momentjs.com/[MomentJS]
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|MomentJS Format with which to bucket, see the https://momentjs.com/docs/#/displaying/[MomentJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[formatnumber_fn]]
+=== `formatnumber`
+
+Turns a `number` into a `string` using a NumeralJS format.
+
+*Accepts:* `number`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `format`
+|`string`
+|NumeralJS format string, see the http://numeraljs.com/#format[NumeralJS documentation]
+|===
+
+*Returns:* `string`
+
+
+[float]
+[[g_fns]]
+== G
+[[getCell_fn]]
+=== `getCell`
+
+Fetches a single cell in a table.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Aliases: `column`, `c`
+|`string`
+|The name of the column value to fetch, defaults to the first column
+
+|`row` 
+
+Alias: `r`
+|`number`
+|The row number, starting at 0 
+
+Default: `0`
+|===
+
+*Returns:* Depends on the data in the cell
+
+
+[float]
+[[gt_fn]]
+=== `gt`
+
+Returns whether the _context_ is greater than the argument.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[gte_fn]]
+=== `gte`
+
+Returns whether the _context_ is greater than or equal to the argument.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[h_fns]]
+== H
+[[head_fn]]
+=== `head`
+
+Gets the first N rows from the `datatable`. See also <<tail_fn>>.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ 
+
+Alias: `count`
+|`number`
+|Return this number of rows from the beginning of the `datatable` 
```

Comment:
How about:

`The number of rows returned from the beginning of the `datatable.`